### PR TITLE
Add home page

### PR DESCRIPTION
**Pull request by Hector096**
1: Added Home Page with a heading.
2: Added gitignore file.
3: Added linters to github/workflows

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
```
Comment (Emmyn5600) [inline .github/workflows/linters.yml:48]: Nice you have included the linters.yml file properly, however there are missing some files in the configuration: for more details check this [template](https://github.com/microverseinc/linters-config/tree/master/html-css)

- .hintrc

- .stylelintrc.json

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: pink;
+}
```
Comment (Emmyn5600) [inline styles.css:3]: - [ ] Please add the .gitignore file to keep the following:

- node_modules
- test.md

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @Hector096 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
Changes required :writing_hand:  

 * #### Please check the comments under the review too
     
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by AshakaE — APPROVED** [review]
### Hi @Hector096  🙋‍♀️, 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**